EventFilter/CastorRawToDigi/src/ZdcUnpacker.cc

```diff
@@ -12,236 +13,278 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include <map>
 
-namespace ZdcUnpacker_impl {
+
+namespace HcalUnpacker_impl {
```

Comment:
This is a word-by-word copy from the HcalUnpacke from
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/EventFilter/HcalRawToDigi/src/HcalUnpacker.cc#L47
to L109.
If you like, I can change it to ZdcUnpacker_impl.
The previous PR copied from L13 to L43, which is used for the old data format.